test/fixtures/host_genomes.yml

```diff
@@ -1,17 +1,29 @@
 one:
   name: host genome 1
+  default_background: real_background
+  s3_bowtie2_index_path: s3://czbiohub-infectious-disease/references/human/bowtie2_genome.tar.gz
+  s3_star_index_path: s3://czbiohub-infectious-disease/references/human/STAR_genome.tar.gz
 
 two:
   name: host genome 2
   metadata_fields: [collection_date, sample_type, sex, age, admission_date, nucleotide_type, water_control]
 
 human:
   name: Human
+  default_background: real_background
   metadata_fields: [collection_date, sample_type, sex, age, admission_date, nucleotide_type, core_field, water_control]
+  s3_bowtie2_index_path: s3://czbiohub-infectious-disease/references/human/bowtie2_genome.tar.gz
```

Comment:
these S3 paths are obsolete. We've also made a choice to avoid revealing the current ones in the codebase, could we preserve that?

Reply:
For sure, I'll change these. How about `s3://bowtie2_genome.tar.gz` ?

Also, for what it's worth, I copied this string from elsewhere in the codebase and appears in many places.